pyproject.toml

```diff
@@ -14,8 +14,8 @@ dependencies = [
     "huggingface_hub",
     "metatensor-learn >=0.3.2,<0.4",
     "metatensor-operations >=0.3.4,<0.4",
-    "metatensor-torch >=0.7.6,<0.8",
-    "metatomic-torch >=0.1.2,<0.2",
+    "metatensor-torch >=0.8.1,<0.9",
```

Comment:
this also added the `assume_unique` flag for Labels creation, should we also add it where relevant in metatrain?